src/components/header/Header.js

```diff
@@ -1,35 +1,43 @@
-import React from 'react';
+import React, {useContext} from 'react';
 import { NavLink } from 'react-router-dom';
+import { LanguageContext } from "../../context/LanguageContext";
 import headerImage from '../../assets/wall-with-plants.jpg';
 import { ReactComponent as FlagNL } from '../../assets/netherlands.svg';
+import { ReactComponent as FlagES } from '../../assets/spain.svg';
 import content from '../../data/content.json';
 import './Header.css';
 
+
 function Header() {
+  const {activeLanguage, setNlFunction, setEsFunction} = useContext(LanguageContext);
+  console.log("hallo ik ben header");
   return (
     <>
       <header>
         <nav>
           <ul>
             <li>
               <NavLink exact to="/about-us">
-                {content.nl.header.menuItems.aboutUs}
+                {content[activeLanguage].header.menuItems.aboutUs}
               </NavLink>
             </li>
             <li>
               <NavLink exact to="/all-plants">
-                {content.nl.header.menuItems.allPlants}
+                {content[activeLanguage].header.menuItems.allPlants}
               </NavLink>
               </li>
             <li className="language-switch">
-              <p>{content.nl.header.changeTo}</p>
-              <FlagNL />
+              <p>{content[activeLanguage].header.changeTo}</p>
+            </li>>
+            <li className="language-switch svg">
+                <button id="buttonNL"  onClick={setNlFunction}><FlagNL /></button>
+                <button id="buttonES"  onClick={setEsFunction}><FlagES /></button>
```

Comment:
de voor waaarde van deze opdracht was: 
De gebruiker kan de taal wisselen door op de vlag in de header te klikken. De taal wordt dan aangepast doormiddel van context. Als de huidige taal op 'Spaans' staat, ziet de gebruiker 'Wissel taal naar [nederlandse vlag]'. Staat de taal op 'Nederlands', dan staat er: 'Cambiar el idioma a [spaanse vlag]'.

hier staan beide vlaggen in de header, je kan dit met een turnery operator oplossen; 
 {activeLanguage === 'nl' ? *<*FlagEs /*>* : *<*FlagNL /*>*}

Reply:
Gelijk heb je! Dit was de opdracht. Was ook half op weg met met turnery operator // op plaats van flag tussen () had ik de gehele button elementen gezet en dit werkte niet!// dit op te lossen. Still some practice improvement in combinatie met het destructuren in componenten.  Jouw methode werkt in combinatie met de if else statement.

Reply:
Wel zie je in bovenstaande context het gebruik van de Flags vaak door middel van andere methodieken. Bijvoorbeeld een selectbar waar je de flag uit kunt selecteren. Deze methode werkt dan ook alleen wanneer er 2 talen zijn. Bij meerdere talen wordt het alweer anders.